services/src/main/java/com/google/bos/iot/core/bambi/model/SourceRepository.java

```diff
@@ -0,0 +1,141 @@
+package com.google.bos.iot.core.bambi.model;
```

Comment:
Is this specific to Bambi?  Seems like it should be down in the utils or common?